instrumentation/opentelemetry-instrumentation-starlette/tests/test_starlette_instrumentation.py

```diff
@@ -230,7 +230,8 @@ def test_basic_post_request_metric_success(self):
                         dict(point.attributes), expected_duration_attributes
                     )
                     if metric.name == "http.server.duration":
-                        self.assertAlmostEqual(duration, point.sum, delta=30)
```

Comment:
latest had trouble with this assertion because it initializes middleware lazily, so the `duration` ends up being way longer than the actual request's duration. I think in principal, it's only safe to check the request duration is not larger than the test duration rather than comparing the actual values so I changed the assertions. 

I wanted to use `self.assertGreater(point.sum, 0)` but the old ms metric is `round`ed and easily `0` on a fastish machine - not sure if the rounding is actually required for it but that would be a change in the ASGI middleware